app/views/devise/sessions/new.html.erb

```diff
@@ -19,8 +36,16 @@
   <% end %>
 
   <div class="actions">
-    <%= f.submit "Log in" %>
+    <%= f.submit "Log in"  , class: 'submit-btn'%>
   </div>
-<% end %>
+<%end%>
 
 <%= render "devise/shared/links" %>
+
+
+</div>
+</div>
+</body>
+</html>
```

Comment:
With Rails, we don't need to write these HTML tags. Rails view files will automatically include them. Kindly remove these lines.